src/Core__Date.resi

```diff
@@ -35,16 +39,132 @@ type localeOptions = {
   timeZoneName?: [#long | #short],
 }
 
-@send external valueOf: t => time = "valueOf"
-@new external make: unit => t = "Date"
-@new external fromString: string => t = "Date"
-@new external fromTime: time => t = "Date"
-@new external makeWithYM: (~year: int, ~month: int) => t = "Date"
-@new external makeWithYMD: (~year: int, ~month: int, ~date: int) => t = "Date"
-@new external makeWithYMDH: (~year: int, ~month: int, ~date: int, ~hours: int) => t = "Date"
+/** 
+`valueOf(date)`
```

Comment:
Once this is in main I'll actually remove this function. Not needed as we have `getTime` already and they're the same apparently.